### PR TITLE
OSDOCS-10659: 4.12.58 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.12

Issue:
[OSDOCS-10659](https://issues.redhat.com/browse/OSDOCS-10659)

Link to docs preview:
https://76588--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-58

QE review:
N/A

Additional information:
URLs will return 404 until release goes live (5/30/24)

### CODE REVIEW

**Comment by tmalove** [comment]
Hi @obrown1205  I am attaching the direct preview link here, but you might want to add it in comment 0 too. Thanks!
/remove-label peer-review-in-progress
/label peer-review-done
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by obrown1205** [comment]
> Hi @obrown1205 I am attaching the direct preview link here, but you might want to add it in comment 0 too. Thanks! https://76588--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-58
> 
> /remove-label peer-review-in-progress /label peer-review-done

So sorry about that! Thank you for catching and linking it :)

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -4709,4 +4709,46 @@ $ oc adm release info 4.12.57 --pullspecs
...
+==== Known issue
+
+* Sometimes, the Console Operator status can get stuck in a failed state when the Operator is removed. To work around this issue, patch the Console controller to reset all conditions to the default state when the Console Operator is removed. For example, log in to the cluster and run the following command:
+
+[source,terminal]
+----
+$ oc patch console.operator.openshift.io/cluster --type='merge' -p='{"status":{"conditions":null}}'
+----
+
+(link:https://issues.redhat.com/browse/OCPBUGS-33386[*OCPBUGS-33386*])
```
Comment (adellape) [inline release_notes/ocp-4-12-release-notes.adoc:4749]:
```suggestion
* Sometimes, the Console Operator status can get stuck in a failed state when the Operator is removed. To work around this issue, patch the Console controller to reset all conditions to the default state when the Console Operator is removed. For example, log in to the cluster and run the following command:
+
[source,terminal]
----
$ oc patch console.operator.openshift.io/cluster --type='merge' -p='{"status":{"conditions":null}}'
----
+
(link:https://issues.redhat.com/browse/OCPBUGS-33386[*OCPBUGS-33386*])
```

**Pull request merged**